packages/flutter_tools/templates/app/android-java.tmpl/app/build.gradle.tmpl

```diff
@@ -38,12 +38,6 @@ android {
         targetSdkVersion 28
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
-        {{#androidX}}
```

Comment:
FWIW, Android Studio adds these dependencies to the default template. I wonder if removing them from the Flutter template could cause unexpected behavior when using Android Studio.

Reply:
I'll try locally - I wonder if it will just add them in automatically.

Reply:
Android studio seems fine with this, and I'm able to build the apk/aab with this template.

However, the tests flaked out now on getting the kotlin dependencies...